### PR TITLE
add gha to test example-get-started

### PR DESCRIPTION
**Pull request by dberenbaum**
Step 1 in trying to setup some automation of this repo. This PR adds only an action to test that `example-get-started/generate.sh` runs successfully when changes to `example-get-started` are pushed. Next would be to make another action to do all the deployment steps when a PR is merged, and then we could add similar actions for `example-get-started-experiments`.

### CODE REVIEW

File: .github/workflows/example-get-started-test.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+        cache: 'pip'
+    - uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-region: us-east-2
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
```
Comment (dberenbaum) [inline .github/workflows/example-get-started-test.yaml:21]: Added to repo secrets

**Review by daavoo — APPROVED** [review]
Let's also do example-get-started-experiments (needs CML runner)

File: .github/workflows/example-get-started-test.yaml (hunk 1 above)
Comment (daavoo) [reply .github/workflows/example-get-started-test.yaml:21]: Is it the sandbox I assume?

Comment (daavoo) [reply .github/workflows/example-get-started-test.yaml:21]: Wouldn't you need to add this repo in https://github.com/iterative/itops/blob/dd03f7752ee61f09487902ce1e4e1d1074393748/terraform/iterative/iam_sandbox.tf#L38 ?

Comment (dberenbaum) [reply .github/workflows/example-get-started-test.yaml:21]: Yup, thanks. I was sure I missed something (probably more). That's why I'm starting small.

File: .github/workflows/example-get-started-test.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: example-get-started test
+on: 
+  push:
+    paths:
+      - example-get-started/**
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-latest
```
Comment (daavoo) [inline .github/workflows/example-get-started-test.yaml:9]:
```suggestion
    runs-on: ubuntu-latest
    environment: aws
```

Comment (daavoo) [reply .github/workflows/example-get-started-test.yaml:9]: I think is needed for iterative/itops#2308 to work

**Pull request merged**